stats/stats_test.go

```diff
@@ -720,7 +720,46 @@ func TestNaN(t *testing.T) {
 	}
 }
 
+func TestBucketLookUp(t *testing.T) {
+	var tests = []struct {
+		inputRecord   float64 // input
+		expectedIndex int     // expected
+
+	}{
+		{inputRecord: 999, expectedIndex: 43},
```

Comment:
let's not do a complete whitebox test here, it's quite impossible for a human to know if 43 is the right bucker, use HistogramData and check the start and/or end of the first bucket should be as I mentioned 900-1000 etc etc

Reply:
When I add one record, it doesnt show the start and end as 900 and 1000 instead it only shows the record value which I added as start and end. For solution, I changed expectedIndex with startRecord. For example, I added two record which are 999 and 900,0004. 900,0004 is the start of the bucket. So when I add these two values, I expect from hData to create only one data.

Reply:
it is tuned to reduce buckets that contain either min or max you can add "1" and/or "1000000" to get the expected bucket span